### PR TITLE
feat: add dataset_preview schema

### PR DESCRIPTION
**Pull request by danieljhegeman**
### Reviewers
**Functional:** 
@Bento007 
**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

File: backend/open_api_spec.yml [hunk 1]
```diff
@@ -335,6 +335,57 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/gene_set"
+    collection_with_dataset_preview:
+      type: object
+      required:
```
Comment (Bento007) [inline backend/open_api_spec.yml:340]: These are all fields in a response. You don't need required.

File: backend/open_api_spec.yml [hunk 2]
```diff
@@ -410,6 +461,31 @@ components:
           $ref: "#/components/schemas/distribution"
         schema_version:
           type: string
+    dataset_preview:
+      type: object
+      required:
```
Comment (Bento007) [inline backend/open_api_spec.yml:466]: This is only a response, you don't need required.

**Review by Bento007 — CHANGES_REQUESTED** [review]
What Endpoint with these be used?

**Comment by danieljhegeman** [comment]
outdated

**Pull request closed without merging**